docs/rp2/quickref.rst

```diff
@@ -156,14 +189,15 @@ Software SPI (using bit-banging) works on all pins, and is accessed via the
 Hardware SPI bus
 ----------------
 
-Hardware SPI is accessed via the :ref:`machine.SPI <machine.SPI>` class and
-has the same methods as software SPI above::
+The RP2040 has 2 hardware SPI buses which is accessed via the
+:ref:`machine.SPI <machine.SPI>` class and has the same methods as software
+SPI above::
 
     from machine import Pin, SPI
 
-    spi = SPI(1, 10000000)
-    spi = SPI(1, 10000000, sck=Pin(14), mosi=Pin(13), miso=Pin(12))
-    spi = SPI(2, baudrate=80000000, polarity=0, phase=0, bits=8, firstbit=0, sck=Pin(18), mosi=Pin(23), miso=Pin(19))
+    spi = SPI(1, 10000000)  # Default assignment: sck=Pin(10), mosi=Pin(11), miso=Pin(8) 
```

Comment:
Just a suggestion, but MicroPython supports underscores in numeric literals which improves readability particularly with large numbers:

```suggestion
    spi = SPI(1, 10_000_000)  # Default assignment: sck=Pin(10), mosi=Pin(11), miso=Pin(8) 
```